_docs/withdrawal/execution.md

```diff
@@ -30,7 +30,7 @@ yourself from your smartphone previously.
     <span class="warning">transaction ID</span> for your withdrawal; make a
     note of it.
 4. Verify the withdrawal on the public blockchain.
-    1. Go to [Blockr](https://www.coinbase.com/) , paste
+    1. Go to [Blockstream](https://blockstream.info/) , paste
     the <span class="warning">destination address</span> into the search bar,
     and press Enter. You'll be taken to a page that says "Bitcoin Address" at
     the top, with the <span class="warning">destination address</span> listed underneath.
```

Comment:
This should have the same wording changes as the deposit execution ("Bitcoin Address" to "Address", "Balance" to "Confirmed Unspent")

Reply:
changed to "confirmed spent" since this is for withdrawing funds, though either value would work.

Reply:
We are looking at the destination address here, not the cold storage address. It should be confirmed UNspent.

Then again, the recipient might have spent it already, so this isn't an entirely reliable process.